setup.cfg

```diff
@@ -176,3 +193,24 @@ allow_untyped_defs = False
 
 [mypy-gtc.*]
 allow_untyped_defs = False
+
+
+#-- pytest --
+[tool:pytest]
+# Set coverage options but do not run it by default (only tox does)
+addopts = --cov-config=setup.cfg --cov-report html
+markers =
+    lit_suite: pytest item running a LLVM-lit test suite
+norecursedirs =
+    dist
+    build
+    _local*
+    .*
+testpaths = tests
+
+
+#-- sphinx --
```

Comment:
commented on purpose?

Reply:
Yes, this data is out of sync until we change the repo structure to have a distribution package per subproject.